apps/web/styles/app.css

```diff
@@ -59,9 +57,8 @@ body,
 }
 
 #composer {
-  position: sticky;
-  bottom: 0;
-  left: 0;
+  position: absolute;
```

Comment:
Have you tested this? Does it behave properly when you scroll down the messages?

Reply:
Can you elaborate on the proper behavior, pleases?

Reply:
Does it stay at the bottom when you scroll?

Reply:
Yes. You can try it out locally as well. 🙂